types/bigbite__wp-cypress/package.json

```diff
@@ -0,0 +1,6 @@
+{
+    "private": true,
+    "dependencies": {
+        "cypress": "*"
```

Comment:
can this be a specific version? preffereably latest one?

Reply:
@peterblazejewicz Sure thing! Specified the `cypress` version as `^12.3.0` (latest) in e36d6b6, and re-ran tests (`npm test @bigbite/wp-cypress`).

Additionally, the project I'm actually using my types on, also uses the latest version of `cypress`.